.helm/data-claims-api/values/staging.yaml

```diff
@@ -12,21 +12,21 @@ image:
   tag: null
 
 service_account:
-  name: laa-data-stewardship-civil-claims-api-staging-service-account
+  name: laa-data-claims-api-staging-service-account
 
 service:
   type: ClusterIP
   port: 8080
 
 ingress:
   annotations:
-    external-dns.alpha.kubernetes.io/set-identifier: "laa-data-stewardship-civil-claims-api-dsp-chart-laa-data-stewardship-civil-claims-api-staging-green"
+    external-dns.alpha.kubernetes.io/set-identifier: "laa-data-claims-api-dsp-chart-laa-data-claims-api-staging-green"
```

Comment:
Could we use the same chart name as the chart directory, i.e `data-claims-api` instead of `dsp-chart`